src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -300,8 +300,8 @@ def save_as_terraform_definitions_on_workspace(self, installation: Installation)
 
 
 class Mounts(CrawlerBase[Mount]):
-    def __init__(self, backend: SqlBackend, ws: WorkspaceClient, inventory_database: str):
-        super().__init__(backend, "hive_metastore", inventory_database, "mounts", Mount)
+    def __init__(self, backend: SqlBackend, ws: WorkspaceClient, inventory_catalog: str, inventory_database: str):
```

Comment:
Note that it is called `inventory_catalog` and `inventory_database` here where the others have just `catalog` and `schema`, maybe clean that up